tracing/opentracing/endpoint.go

```diff
@@ -3,10 +3,9 @@ package opentracing
 import (
 	"context"
 
+	"github.com/go-kit/kit/endpoint"
```

Comment:
please revert change in import grouping, we typically do:

1. standard library
2. external dependencies
3. go-kit internal dependencies